src/Microsoft.ML.TimeSeries/SrCnnEntireAnomalyDetector.cs

```diff
@@ -625,7 +649,7 @@ private double PredictNext(double[] data)
                 return (data[1] + slopeSum);
             }
 
-            private void AverageFilter(double[] data, int n)
+            private void AverageFilter(double[] data, int n, bool ignoreFirst=false)
```

Comment:
is `ignoreFirst` ever used?

Reply:
Removed this parameter, it is unused actually.

---
In reply to: [527820832](https://github.com/dotnet/machinelearning/pull/5502#discussion_r527820832) [](ancestors = 527820832)